### PR TITLE
Bug fix, quiz question viewer

### LINKED ISSUE #187: Module Editor: file upload issues
When editing a module and the user does not want to change the file currently uploaded the user has to reupload the file currently in the system and the input field is "required"

Currently working on a fix will update here with new information

### PR DESCRIPTION
**Pull request by Michael-Dews**
This PR is making the following changes to dev:

1: Fixing a number of bugs in the quiz creator, such as you no longer need to close and re-open the question viewer for the list to update in the viewer.

2: Removed My Files button from the sidebar. Allowed Admin level users access to the creation tools.

3: Fixed a number of alert typos.

4: Fixed a bug in the module creator where creating a quiz module failed to redirect you to the course page

5: Tweaked the way the module editor handles uploads so that if you are editing a module with a file that has been uploaded: PDF, File, or Video. As long as the module type remains the same, you are not required to upload the file again. With no fil;e submission in this case the old file will still be displayed.

### CODE REVIEW

**Review by dmols — APPROVED** [review]
Everything works. Good to push once more reviews come in.

**Comment by collyntheshots** [comment]
What is this PR even for?

**Review by dmols — APPROVED** [review]
Number of questions to pass still allows negative values. I'd say if there's no way to limit the options to 1 and above, make the component a textbox instead, if it's not too much trouble.

**Review by dmols — CHANGES_REQUESTED** [review]
Same as comment above; I just accidentally approved instead of requesting change.

**Review by collyntheshots — APPROVED** [review]
Everything seems to be in working order.
I dont know if this is intentional based off the descriptions but I had a PDF module that had been made, I then went to edit the module and changed the "type" to file but did not submit. I then changed it back to PDF and did not need to add a new PDF file to submit my edit

File: frontend/toolkit-webapp/src/pages/ManageMyCourses.js [hunk 1]
```diff
@@ -135,16 +148,46 @@ const ManageMyCourses = (props) => {
...
+                                                    <Button className={classes.dialogButton2}  size="small" variant="contained" type='submit' onClick={handleCloseDialog} >
+                                                        No
+                                                    </Button>
+                                               
+                                            </DialogContent>
+                                        </Dialog>
                                     </div>
-                                </Link>
 
+                                    : null} */}
```
Comment (mbrancato31) [inline frontend/toolkit-webapp/src/pages/ManageMyCourses.js:190]: if you dont need this, then delete it

**Review by mbrancato31 — APPROVED** [review]

**Pull request merged**